From b42ea59f5e94237a3979415bb244efca78888378 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Fri, 4 Aug 2017 18:31:36 +0800
Subject: [PATCH 2329/5242] MLK-16136-8 ARM64: dts: imx8mq: add 'endian'
 property for irqsteer

commit  8deff68d00095549e625593ca103cc32e45c32ce from
https://source.codeaurora.org/external/imx/linux-imx.git

The relationship between an interrupt and its
register and bit offset has two possible cases:

    .Littel Endian (LSB)
    .Big Endian (MSB)

So add 'endian' property to help irqsteer kernel
driver to configure interrupts correctly.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index 0af8ea3..e0fbdfb 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -314,6 +314,7 @@
 		interrupt-parent = <&gpc>;
 		#interrupt-cells = <2>;
 		nxp,irqsteer_chans = <2>;
+		nxp,endian = <1>;		/* MSB */
 		clocks = <&clk IMX8MQ_CLK_DUMMY>;
 		clock-names = "ipg";
 	};
-- 
1.7.9.5

